mplleaflet/_display.py

```diff
@@ -139,8 +148,25 @@ def display(fig=None, closefig=True, **kwargs):
     if closefig:
         plt.close(fig)
 
-    html = fig_to_html(fig, template="ipynb.html", **kwargs)
-    return HTML(html)
+    html = fig_to_html(fig, **kwargs)
+
+    # We write explicitely the CSS and the JS into the html string.
+    html = html\
+        .replace('<link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.7.1/leaflet.css" />',
+                 '<style>{}</style>'.format(_leaflet_css))\
+        .replace('<script src="http://cdn.leafletjs.com/leaflet-0.7.1/leaflet.js"></script>',
+                 '<script>{}</script>'.format(_leaflet_js))
```

Comment:
@BibMartin The `.replace()` call can be removed now that you've included it explicitly in the template, right?